doc/source/management/data_at_rest_encryption.rst

```diff
@@ -154,34 +273,146 @@ Turning this option off at runtime makes server to create all subsequent
+
+.. code-block:: guess
+
+   mysql> CREATE TABLE ... ENCRYPTION=’KEYRING’ ENCRYPTION_KEY_ID=NEW_ID
+
+**NEW_ID** is an unsigned 32-bit integer that refers to the numerical part of
+the ``percona_innodb-`` key.  When you assign a numerical identifer in the
+``ENCRYPTION_KEY_ID`` clause, the server uses the latest version of the
+corresponding key. For example, the clause ``ENCRYPTION_KEY_ID=2`` refers to the
+latest version of the ``percona_innodb-2`` key from the keyring.
+
+In case the ``percona-innodb-`` key with the requested ID does not exist in the
+keyring, |Percona Server| will create it with version 1. If a new
+``percona-innodb-`` key cannot be created with the requested ID, the whole
+``CREATE TABLE`` statement fails
+
+.. rubric:: FORCE_KEYRING
+	    
+New tables are created encrypted and keyring encryption is enforced.
+
+.. rubric:: ONLINE_TO_KEYRING
+
+All tables created or altered without the ``ENCRYPTION=NO`` clause 
+are encrypted with the latest version of the default encryption key. If a table
+being altered is already encrypted with the master key, the table is recreated
+encrypted with the latest version of the default encryption key.
+
+.. rubric:: ONLINE_TO_KEYRING_FORCE
+
+It is only possible to apply the keyring encryption when creating or altering
+tables.
+
+.. note::
+
+   The ``ALTER TABLE`` statement changes the current encryption mode only if you
+   use the ``ENCRYPTION`` clause.
+
+.. seealso::
+
+   |MariaDB| Documentation: ``innodb_encrypt_tables`` Option
+      https://mariadb.com/kb/en/library/xtradbinnodb-server-system-variables/#innodb_encrypt_tables
+
+.. variable:: innodb_online_encryption_threads
+
+   :version 5.7.23-24: Implemented
+   :cli: ``--innodb-online-encryption-threads``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Numeric
+   :default: 1
+
+This variable works in combination with the :variable:`innodb_encrypt_tables`
+variable set to ``ONLINE_TO_KEYRING`` and configures the number of threads for
+background encryption. For the online encryption to work, this variable must
+contain a value greater than **zero**.
+
+.. variable:: innodb_online_encryption_rotate_key_age
+
+   :version 5.7.23-24: Implemented
+   :cli: ``--innodb-online-encryption-rotate-key-age``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Numeric
+   :default: 1
+
+By using this variable, you can re-encrypt the table encrypted using
+KEYRING. The value of this variable determines how frequently the encrypted
+tables should be encrypted again. If it is set to **1**, the encrypted table is
+re-encrypted on each key rotation. If it is set to **2**, the table is encrypted
+on every other key rotation.
+      
+.. _data-at-rest-encryption.undo-tablespace:
+
+InnoDB Undo Tablespace Encryption
+================================================================================
+
+.. TODO: More information is needed
+
+The encryption of InnoDB Undo tablespaces is only available when using separate
+undo tablespaces.
```

Comment:
As without separate undo tablespaces, the undo log is part of the system tablespace, and it's encryption is controlled by the system tablespace encryption option.

And should also be alpha quality?

Reply:
It is alpha

Reply:
Status updated as requested